blocklylib-core/src/main/java/com/google/blockly/android/FlyoutUiInterface.java

```diff
@@ -0,0 +1,49 @@
+package com.google.blockly.android;
+
+import android.support.annotation.Nullable;
+
+import com.google.blockly.android.control.BlocklyController;
+import com.google.blockly.android.ui.FlyoutCallback;
+import com.google.blockly.model.FlyoutCategory;
+
+/**
+ * An interface that specifies the actions that can be taken on a flyout ui component.
+ */
+public interface FlyoutUiInterface {
```

Comment:
BlockListUI, since the drawer / flyout behavior is now an implementation detail.